theme/static/css/all.css

```diff
@@ -1 +1 @@
a198}.highlight .sd{color:#2aa198}.highlight .s2{color:#2aa198}.highlight .se{color:#cb4b16}.highlight .sh{color:#2aa198}.highlight .si{color:#cb4b16}.highlight .sx{color:#2aa198}.highlight .sr{color:#2aa198}.highlight .s1{color:#2aa198}.highlight .ss{color:#2aa198}.highlight .bp{color:#268bd2;font-weight:bold}.highlight .vc{color:#268bd2}.highlight .vg{color:#268bd2}.highlight .vi{color:#268bd2}.highlight .il{color:#2aa198}code{padding:.1em .25em;border-radius:.25em;background:#f2ede1}#colorbox,#cboxOverlay,#cboxWrapper{position:absolute;top:0;left:0;z-index:9999;overflow:hidden}#cboxWrapper{max-width:none}#cboxOverlay{position:fixed;width:100%;height:100%}#cboxMiddleLeft,#cboxBottomLeft{clear:left}#cboxContent{position:relative}#cboxLoadedContent{overflow:auto;-webkit-overflow-scrolling:touch}#cboxTitle{margin:0}#cboxLoadingOverlay,#cboxLoadingGraphic{position:absolute;top:0;left:0;width:100%;height:100%}#cboxPrevious,#cboxNext,#cboxClose,#cboxSlideshow{cursor:pointer}.cboxPhoto{float:left;margin:auto;border:0;display:block;max-width:none;-ms-interpolation-mode:bicubic}.cboxIframe{width:100%;height:100%;display:block;border:0;padding:0;margin:0}#colorbox,#cboxContent,#cboxLoadedContent{box-sizing:content-box;-moz-box-sizing:content-box;-webkit-box-sizing:content-box}#cboxOverlay{background:#000}#colorbox{outline:0}#cboxBottomCenter{height:36px}#cboxContent{border-radius:.5em;overflow:visible;background:#fff}.cboxIframe{background:#fff}#cboxError{padding:50px;border:1px solid #ccc}#cboxLoadedContent{border-radius:.5em;padding:1em;border:1px solid #000}#cboxTitle{position:absolute;bottom:-25px;left:0;text-align:center;width:100%;font-weight:bold;color:#ddd}#cboxCurrent{font-size:.8em;position:absolute;bottom:0;left:1.5rem;line-height:1rem;color:#aaa}#cboxPrevious,#cboxNext,#cboxSlideshow,#cboxClose{position:absolute;bottom:-32px;margin:0;padding:0;border:0;overflow:visible;line-height:0;color:transparent;font-size:0;background:transparent}#cboxPrevious::before,#cboxNext::before,#cboxSlideshow::before,#cboxClose::before{content:'~';display:block;height:24px;width:24px;font-size:24px;font-weight:bold;line-height:21px;text-align:center;color:#000;background:#fff;border-radius:100%;border:2px solid #000;box-shadow:0 0 2px #444}#cboxPrevious:hover::before,#cboxNext:hover::before,#cboxSlideshow:hover::before,#cboxClose:hover::before{border-color:#fff;color:#fff;background:#000}#cboxPrevious{left:0}#cboxPrevious::before{content:'‹'}#cboxNext{right:0}#cboxNext::before{content:'›'}#cboxSlideshow::before{content:'S'}#cboxClose{top:4px;right:4px;bottom:auto}#cboxClose::before{content:'×'}#cboxPrevious:active,#cboxNext:active,#cboxSlideshow:active,#cboxClose:active{outline:0}.cboxSlideshow_on #cboxSlideshow{background-position:-125px 0;right:27px}.cboxSlideshow_on #cboxSlideshow:hover{background-position:-150px 0}.cboxSlideshow_off #cboxSlideshow{background-position:-150px -25px;right:27px}.cboxSlideshow_off #cboxSlideshow:hover{background-position:-125px 0}
```

Comment:
No, the point is that you shouldn't be editing this file directly; you should edit `all.scss`, then use a Sass compiler to generate this file.